functions/java-jre.sh

```diff
@@ -18,27 +18,30 @@ java_webupd8_archive() {
 }
 
 java_zulu() {
-  local FILE="/var/tmp/.zulu.$$"
-  local INSTALLROOT=/opt/jdk
-  local TEMPROOT=/opt/jdk-new
+  local FILE
+  local INSTALLROOT
+  local TEMPROOT
+  local JAVA
+  FILE="/var/tmp/.zulu.$$"
+  INSTALLROOT=/opt/jdk
+  TEMPROOT=/opt/jdk-new
   mkdir ${INSTALLROOT}
   mkdir ${TEMPROOT}
-  
+
   if is_arm; then
     # Latest version check https://www.azul.com/downloads/zulu-embedded
-    local JAVA=zulu8.33.0.134-jdk1.8.0_192-linux_aarch32hf
+    JAVA=zulu8.33.0.134-jdk1.8.0_192-linux_aarch32hf
   else
     # Latest version check https://www.azul.com/downloads/zulu-linux
-    local JAVA=zulu8.33.0.1-jdk8.0.192-linux_x64
+    JAVA=zulu8.33.0.1-jdk8.0.192-linux_x64
   fi
   whiptail --textbox $BASEDIR/includes/azul-zulu-license.md --scrolltext 27 116
-  
+
   cond_redirect wget -nv -O $FILE http://cdn.azul.com/zulu-embedded/bin/${JAVA}.tar.gz
   cond_redirect tar -xpzf $FILE -C ${TEMPROOT}
   if [ $? -ne 0 ]; then echo "FAILED (Zulu java)"; rm -f ${FILE}; exit 1; fi
-  rm -rf $FILE ${INSTALLROOT}/*
+  rm -rf $FILE ${INSTALLROOT:?}/*
```

Comment:
Could you explain this change?

Reply:
Since this is changed by the commit for SC2115:
See https://github.com/koalaman/shellcheck/wiki/SC2115
If for any reason $INSTALLROOT would ever be an empty string, this would delete our root file system.
This is just a safeguard against this.
I wasn't aware of this problem until shellcheck told me about it.

Reply:
oooh would you look at that. Nice catch Mr Bot 😄